src/main/java/uk/gov/hmcts/ccd/data/casedetails/search/builder/GrantTypeQueryBuilder.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.ccd.data.casedetails.search.builder;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.hmcts.ccd.domain.model.casedataaccesscontrol.RoleAssignment;
+
+public interface GrantTypeQueryBuilder {
+
+    String QUERY_WRAPPER = "( %s )";
+
+    String QUERY = "%s in (:%s)";
+
+    String EMPTY = "";
+
+    String SECURITY_CLASSIFICATION = "security_classification";
+
+    String JURISDICTION = "jurisdiction";
+
+    String REFERENCE = "reference";
+
+    String CASE_TYPE_ID = "case_type_id";
+
+    String LOCATION = "data" + " #>> '{caseManagementLocation,BaseLocationId}'";
+
+    String REGION = "data" + " #>> '{caseManagementLocation,RegionId}'";
```

Comment:
```suggestion
    String LOCATION = "data" + " #>> '{caseManagementLocation,baseLocation}'";

    String REGION = "data" + " #>> '{caseManagementLocation,region}'";
```
For the updated changes 🙏